lib/rust/profiler-macros/src/lib.rs

```diff
@@ -0,0 +1,303 @@
+#![feature(proc_macro_span)]
```

Comment:
No module doc. Probably the linter of this crate is not configured a linter of other crates – please copy the configuration from other `lib.rs`. Unfortunately, it is impossible to create a single linter configuration for all the crates :(